cli.go

```diff
@@ -60,13 +79,30 @@ func main() {
 	}
 }
 
-func PromptDiscord(action string) *DiscordInstall {
+func PromptDiscord(action string, client *DiscordInstall, dir string, branch string) *DiscordInstall {
```

Comment:
The client param seems unused and you always pass nil to it. Please remove it if unused

```suggestion
func PromptDiscord(action, dir, branch string) *DiscordInstall {
```